src/libsync/theme.h

```diff
@@ -160,9 +160,16 @@ class OWNCLOUDSYNC_EXPORT Theme : public QObject
      * Setting a value here will pre-define the server url.
      *
      * The respective UI controls will be disabled
+     * Deprecated: Use overrideServerUrlV2 as it allows overrides
      */
+    Q_DECL_DEPRECATED_X("Use overrideServerUrlV2")
     virtual QString overrideServerUrl() const;
```

Comment:
What's the incentive to keep this function rather than changing it directly?

Reply:
The reason is that the auto generated themes define this function...

Reply:
I see.

I suppose the `V2` suffix is the reason for this confusion. The name should give an idea what the method does. This one is just a shallow wrapper that allows overrides via an environment variable, so I guess that fact should somehow end up in the method name.